pkg/email/templates/client_requested.go

```diff
@@ -42,18 +42,51 @@ func init() {
 }
 
 func newClientRequestedData(_ context.Context, data email.NotificationTemplateData) (email.NotificationTemplateData, error) {
-	var nData ttnpb.CreateClientRequest
-	if err := ttnpb.UnmarshalAny(data.Notification().GetData(), &nData); err != nil {
+	var dd ttnpb.CreateClientEmailMessage
```

Comment:
Just `data`.

Reply:
changed to `emailMsg`, since the parameter is already named `data`